windows-installer/setup.nsi

```diff
@@ -33,6 +33,9 @@
 ; Directory to install all files into.  This makes uninstall easier and safer.
 !define MAIN_INSTDIR "$INSTDIR\Main"
 
+; Directory for the user to add custom configs.
+!define CUSTOM_CONFIG_DIR "$INSTDIR\config.d"
```

Comment:
What happened to factoring out the config directory? What if you decide to move `fluentd.conf` from `$INSTDIR` to somewhere else? We'd want to move the `config.d` subdirectory as well.